README.md

```diff
@@ -127,4 +127,27 @@ uvicorn main:app --reload
 
 The frontend will be available at `localhost:8000` 
 
+## 🗂️ Using Local Directories
```

Comment:
I think you wrote that before I changed the README in https://github.com/cyclotruc/gitingest/commit/f6ce264720b36e9f014487c1d60cd372497304ce